core/jobs.py

```diff
@@ -233,19 +259,29 @@ def cancel(cls, job_id, user_id):
 
     @classmethod
     def is_active(cls, job_id):
+        """Asserts if job is still active.
+        Args:
+            job_id: str. The id of the job.
+        Returns: Boolean. If the job is active or not."""
```

Comment:
The returns docstring should start on a new line after "Returns:", and `Boolean` should be changed to `bool`.